src/main/proto/th2_grpc_common/common.proto

```diff
@@ -193,12 +194,14 @@ message MessageFilter {
   string direction = 2 [deprecated = true];
   map<string, ValueFilter> fields = 3;
   ComparisonSettings comparison_settings = 4;
+  google.protobuf.StringValue description = 5;
 }
 
 message MetadataFilter {
   message SimpleFilter {
     FilterOperation operation = 1;
     bool key = 2;
+    google.protobuf.StringValue description = 5;
```

Comment:
I am not sure that we should add the description to SimpleFilter. I cannot image the case when we need this